django/contrib/admin/static/admin/css/base.css

```diff
@@ -914,22 +914,37 @@ a.deletelink:focus, a.deletelink:hover {
     text-decoration: none;
 }
 
+#logout-form {
+    display: inline;
+}
+
+#logout-form button {
+    background: none;
+    border: 0;
+    cursor: pointer;
+    font-family: "Roboto","Lucida Grande","DejaVu Sans","Bitstream Vera Sans",Verdana,Arial,sans-serif;
```

Comment:
I wonder if we should pull that into a variable to allow easier changes (like we do with colors)

Reply:
Sound reasonable, let's leave this for a separate cleanup.